src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/CreateNewUserWithRolesTest.java

```diff
@@ -360,4 +401,44 @@ public void should_return_409_when_same_invited_new_user_is_super_user_of_pendin
         assertThat((String) newUserResponse.get("response_body"))
                 .contains("\"errorDescription\":\"409 User already exists\"");
     }
+
+    @Test
+    public void super_user_can_have_caa_roles_fpla_or_iac_roles_not_puiCaa_caseworkerCaa() {
+        userProfileCreateUserWireMockWithExtraRoles();
+        String organisationIdentifier = createOrganisationRequest();
+        updateOrganisation(organisationIdentifier, hmctsAdmin, "ACTIVE");
+
+        List<String> userRoles = new ArrayList<>();
+        userRoles.add(puiUserManager);
+        userRoles.add("caseworker");
+
+        String userIdentifier = retrieveSuperUserIdFromOrganisationId(organisationIdentifier);
+
+        userProfileCreateUserWireMockWithExtraRoles();
+
+        Map<String, Object> newUserResponse =
+                professionalReferenceDataClient.addUserToOrganisationWithUserId(organisationIdentifier,
+                        inviteUserCreationRequest(randomAlphabetic(5) + "@email.com", userRoles),
+                        hmctsAdmin, userIdentifier);
+
+        String id = (String) newUserResponse.get("userIdentifier");
+
+        userProfileCreateUserWireMockWithExtraRoles();
+
+        Map<String, Object> response = professionalReferenceDataClient.findUsersByOrganisationWithReturnRoles(
+                "true", puiCaseManager, id);
+
+        assertThat(response.get("organisationIdentifier")).isNotNull();
+        assertThat(((List<ProfessionalUsersResponse>) response.get("users")).size()).isEqualTo(1);
+        List<HashMap> professionalUsersResponses = (List<HashMap>) response.get("users");
+
+        assertThat(professionalUsersResponses.get(0).get("userIdentifier")).isNotNull();

```

Comment:
If this string literal is being used multiple times, we can convert it to a constant in test file.

Reply:
Addressed here: https://github.com/hmcts/rd-professional-api/pull/961/commits/35ddeb8f43b6dbb8105999455e2eba9cb228b0f9